test/blackbox-tests/test-cases/no_dynlink.t

```diff
@@ -0,0 +1,37 @@
+  $ cat >dune-project <<EOF
```

Comment:
Can you add a few sentences here to describe the test?

Reply:
Fixed, thanks.